### ISSUE TITLE
Perhaps it's not just a downloader!!! virus script??

### ISSUE THREAD

**Issue by songroger** [684102/PornDownloader#32]
Why do you write the script to insert other sites, Google、Youtube、Facebook、github, etc..
If you haven't hijack google title in the last version, I even won't  find the script did so much!!

**Comment by 684102** [comment]
Hello, some people want to know the scripts work, so that title mean script work on that site. 
But on latest version i just turn it off.

**Comment by songroger** [comment]
Yes, The title you just did add a tail.  I just wonder is it necessory to monitor google and other sites for a downloader and did other business?

**Comment by 684102** [comment]
> Yes, The title you just did add a tail. I just wonder is it necessory to monitor google and other sites for a downloader and did other business?

The script by default includes for those domain , but i don't have any plan for support those sites yet, therefore the script enabled with no task. Maybe in future if you have any idea!

ps/ on youtube, porndownloader help you remove ads with simple code, it not using lots of memory such as adblock.

**Comment by songroger** [comment]
Fine, I'll try to review the script later，thank you for your reply.  I just wanna a clean downloader, not some unknown virus task, I think this is also the others do. A bit of  advice, maybe you could remove some unrelated monitoring domains on the next version, even it did or not did something.
